src/saga_action_error.rs

```diff
@@ -65,7 +65,7 @@ pub enum ActionError {
 
     /// The framework failed to deserialize the saga parameters, an action's
     /// successful result, or an action's error.
-    #[error("failed to serialize action's result")]
+    #[error("failed to deserialize action's result")]
```

Comment:
The comment says there are other cases here too.  Want to change this to "failed to deserialize action result, saga parameters, or an action error"?  Better might be something more vague but "failed to deserialize something" doesn't sound great..."failed to deserialize saga log item"?

Reply:
I see this applies to the serialize case.  What if we just say: "serialize failed" and "deserialize failed"?

If you want to add details, I think you can just append ": {message}" or something close to it.